### ISSUE TITLE
Bundler/OrderedGems error is not detected

### ISSUE THREAD

**Issue by YusukeIwaki** [prontolabs/pronto-rubocop#43]
I'm using [Bundle/OrderedGems](https://rubocop.readthedocs.io/en/latest/cops_bundler/#bundlerorderedgems) rule in my project.

When a gem is added to Gemfile, like below:

```
source 'https://rubygems.org'
git_source(:github) { |repo| "https://github.com/#{repo}.git" }

ruby '2.6.3'

gem 'bcrypt', '~> 3.1.7'
gem 'bootsnap', '>= 1.1.0', require: false
gem 'jbuilder', '~> 2.5'
gem 'mysql2', '~> 0.5.2'
gem 'puma', '~> 3.11'
gem 'rails', '~> 5.2.2'
gem 'file_validators' # <-- added line
```

`bundle exec rubocop` shows a error:

```
Gemfile:13:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem file_validators should appear before rails.
gem 'file_validators' # <-- added line
^^^^^^^^^^^^^^^^^^^^^
```

however with pronto-rubocop, it is now shown... 

```
# pronto run --runner=rubocop --commit=<base commit>
```

How can I show error with pronto-rubocop? 🤔 

Thanks

**Comment by doomspork** [comment]
@YusukeIwaki can you confirm the rubocop versions you're using locally and we use in this package are the same?

**Comment by sunny** [comment]
It seems to me that this does not work in the latest RuboCop version since it depends on `pronto`'s `ruby_patches` method which doesn't include `Gemfile`s.

I've pushed a PR against `pronto` which should fix this issue in `pronto-rubocop`: https://github.com/prontolabs/pronto/pull/344

**Comment by doomspork** [comment]
Awesome @YusukeIwaki! Thank you. Your PR looks great. I'll give the other @prontolabs/core folks a day or so to comment before merging it.

**Comment by ashkulz** [comment]
This should be now available with the `0.11.0` release.
